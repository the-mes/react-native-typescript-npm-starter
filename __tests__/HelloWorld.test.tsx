import React from 'react'
import { render } from '@testing-library/react-native'

import { HelloWorld } from '../src'

// TODO define platform

describe('HelloWorld', () => {
  it('should render component with default message', () => {
    const { getByText, toJSON } = render(<HelloWorld />)

    expect(getByText('Hello')).toBeDefined()
    expect(toJSON()).toMatchSnapshot()
  })

  it('should render component with custom message', () => {
    const { getByText, toJSON } = render(<HelloWorld text="Good Morning!" />)

    expect(getByText('Good Morning!')).toBeDefined()
    expect(toJSON()).toMatchSnapshot()
  })
})
