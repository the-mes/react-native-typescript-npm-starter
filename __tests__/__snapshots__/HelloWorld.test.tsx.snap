// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld should render component with custom message 1`] = `
<Text>
  Good Morning!
</Text>
`;
